src/api/app/models/bs_request_action_submit.rb

```diff
@@ -170,6 +170,14 @@ def forward
     forward_object
   end
 
+  def source_srcmd5
+    Package.find_by_project_and_name(source_project, source_package)&.dir_hash({ rev: source_rev }.compact)&.[]('srcmd5')
```

Comment:
See `target_package_object`

Reply:
Doesn't it fall out of sync with source/target project/package after the project/package is removed?

Reply:
`it` being `target_package_object`?

Reply:
Yes

Reply:
It's the *same* code you would re-use so it would behave the same way as it does now, no?

Reply:
Nope, one comes from `source_package` column and the object from `source_package_id` database column, which means this information can fall out of sync